testgrid/pkg/configurator/options/options_test.go

```diff
@@ -61,7 +62,7 @@ func Test_Options(t *testing.T) {
 					JobConfigPathFlagName:                 "prow-job-config",
 					SupplementalProwConfigsFileNameSuffix: "_prowconfig.yaml",
 				},
-				Output: "gs://foo/bar",
+				Output: flagutil.NewStringsBeenSet("gs://foo/bar"),
```

Comment:
Can you add an options test for this new feature?

Reply:
good idea, done